File: mario.c
#include <cs50.h>
#include <stdio.h>

void print_hashmark(int repeat);
void print_new_line(void);
void print_space(int repeat);

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");

        if (height >= 1 && height <= 8)
        {
            for (int i = 1; i <= height; i++)
            {
                print_space(height - i);
                print_hashmark(i);
                print_new_line();
            }
        }
    }
    while (height < 1 || height > 8);   
}

void print_hashmark(int repeat)
{
    for(int i = 0; i <repeat; i++)
    {
        printf("#");
    }
}

void print_new_line(void)
{
    printf("\n");
}

void print_space(int repeat)
{
    for (int i = 0; i < repeat; i++)
    {
        printf(" ");
    }
}

File: mario.py
from cs50 import get_int

heigth = get_int("Heigth: ")

while(heigth < 1 or heigth > 8):
    heigth = get_int("Heigth: ")

for i in range(heigth):
    print(" " * (heigth-i+1), end="")
    print("#" * (i+1), end="")
    print()
